Tests/PointFreeTests/__Snapshots__/InviteEmailTests/testInviteAcceptance.1.html

```diff
@@ -0,0 +1,80 @@
+    </style>
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    </meta>
+    <meta http-equiv="content-type" content="html" charset="utf-8">
+    </meta>
+    <title>
+      Blob has accepted your invitation!
+    </title>
+  </head>
+  <body bgcolor="#FFFFFF" style="height:100%;-webkit-box-sizing:border-box;-moz-box-sizing:border-box;-ms-box-sizing:border-box;-o-box-sizing:border-box;box-sizing:border-box">
+    <span style="color:transparent;display:none;opacity:0.0;height:0;width:0;max-height:0;max-width:0;overflow:hidden">
```

Comment:
boy oh boy are the guts of an inlined HTML email ugly.

but hey, even with all the styles inlined AND the styles rendered into `<style>`, the email is only 17kb.